### PR TITLE
Add Prettier for HTML

### PR DESCRIPTION
**Pull request by katydecorah**
This PR will run prettier on html files during commit. This will:

* Make example code styling consistent.
* Help catch HTML errors:
    - mismatched closing tag (re: #149)
   - extraneous closing div at the end of docs/pages/example/change-case-of-labels.html

### CODE REVIEW

**Comment by katydecorah** [comment]
By the way, this branch is live on staging

File: docs/pages/example/add-3d-model.html [hunk 1]
```diff
@@ -1,93 +1,119 @@
...
+        pitch: 60,
+        antialias: true // create the gl context with MSAA antialiasing, so custom layers are antialiased
+    }));
 
-// parameters to ensure the model is georeferenced correctly on the map
-var modelOrigin = [148.98190, -35.39847];
-var modelAltitude = 0;
-var modelRotate = [Math.PI / 2, 0, 0];
+    // parameters to ensure the model is georeferenced correctly on the map
+    var modelOrigin = [148.9819, -35.39847];
```
Comment (danswick) [inline docs/pages/example/add-3d-model.html]: Heh, prettier does not like sigfigs i guess

File: docs/pages/example/animate-point-along-line.html [hunk 2]
```diff
@@ -1,51 +1,48 @@
...
-        "id": "point",
-        "source": "point",
-        "type": "circle",
-        "paint": {
-            "circle-radius": 10,
-            "circle-color": "#007cbf"
-        }
-    });
+        map.addLayer({
+            id: 'point',
```
Comment (danswick) [inline docs/pages/example/animate-point-along-line.html]: I know this has been a point of contention in the past, but  I _think_ we sorta settled on double quotes for property names and values for Mapbox GL Style objects. 

for ref: https://github.com/mapbox/documentation/issues/18#issuecomment-323869267

File: docs/components/example_code.js [hunk 3]
```diff
@@ -9,9 +9,9 @@ import Note from '@mapbox/dr-ui/note';
 import WarningImage from '@mapbox/dr-ui/warning-image';
 
 const highlightTheme = require('raw-loader!@mapbox/dr-ui/css/prism.css');
-const viewport = `<meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />`;
-const css = `body { margin: 0; padding: 0; }
-        #map { position: absolute; top: 0; bottom: 0; width: 100%; };`;
+const viewport = `<meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />`;
+const css = `\tbody { margin: 0; padding: 0; }
```
Comment (danswick) [inline docs/components/example_code.js]: Oooh, nice! They look so much better now ✨

File: docs/pages/example/animate-point-along-line.html (hunk 2 above)
Comment (katydecorah) [reply docs/pages/example/animate-point-along-line.html]: @danswick Thanks for pointing that out! I added a comment on the thread: https://github.com/mapbox/documentation/issues/18#issuecomment-560944069 (tl;dr Prettier take the wheel)

**Comment by katydecorah** [comment]
> My only real significant concern is around the quotes in literal JSON (Mapbox GL style snippets and GeoJSON in particular).

I added `--quote-props preserve` to the Prettier config for HTML and reran Prettier on the examples and now the quotes are back.

**Pull request merged**